float64[] ur_state
bool gripper_state
---
